blender/arm/exporter.py

```diff
@@ -2496,9 +2496,9 @@ def export_traits(self, bobject, o):
                 if t.enabled_prop == False:
                     continue
                 x = {}
-                if t.type_prop == 'Logic Nodes' and t.nodes_name_prop != '':
+                if t.type_prop == 'Logic Nodes' and t.node_tree_prop != None and t.node_tree_prop.name != '':
```

Comment:
If we want to allow for seamless upgrade from earlier versions of armory we could reference `t.name` here instead of `t.node_tree_prop.name`. Earlier versions of Armory have the `name` property which is set from the node tree name, so it will work on old and new versions without any user interaction. It is up to you @luboslenco what you want to do with that.